code/modules/clothing/head/misc.dm

```diff
@@ -280,5 +280,5 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
I don't see how it could be considered to be messed up grammar, considering the fact you mentally include the strikethrough text and use it as context while reading it. If it was standing alone without that, I could understand your point, but as is it's just a dumb punchline.